Source/Scene/Expression.js

```diff
@@ -670,6 +687,13 @@ define([
         };
     }
 
+    function getEvaluateTernaryFunction(call) {
+        var evaluate = ternaryFunctions[call];
+        return function(feature) {
+            return evaluate(this._left.evaluate(feature), this._right.evaluate(feature));
```

Comment:
Should be:
`return evaluate(this._left.evaluate(feature), this._right.evaluate(feature), this._test.evaluate(feature));`